quick-start-with-tidb.md

```diff
@@ -57,10 +57,10 @@ As a distributed system, a basic TiDB test cluster usually consists of 2 TiDB in
         {{< copyable "shell-regular" >}}
 
         ```shell
-        tiup playground v4.0.0 --db 2 --pd 3 --kv 3 --monitor
+        tiup playground v4.0.11 --db 2 --pd 3 --kv 3 --monitor
```

Comment:
@morgo @King-Dylan Does this need version-specific change in master and release-5.0?

Reply:
That would be ideal. master and v5.0 do not technically have releases yet, so I think we can keep it as is for this merge.